curation_concerns-models/app/models/concerns/curation_concerns/upload_set_behavior.rb

```diff
@@ -5,7 +5,7 @@ module UploadSetBehavior
     include CurationConcerns::Noid
 
     included do
-      has_many :file_sets, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.isPartOf
+      has_many :works, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.isPartOf, class_name: 'ActiveFedora::Base'
```

Comment:
Yes. If you don't it attempts to load classes named `Work`, which doesn't exist.